docs/docs/glossary.md

```diff
@@ -204,6 +212,10 @@ A way of writing HTML content with plain text, using special characters to denot
 
 ## N
 
+### Navigator
+
+A Navigator object represents information about the browser. You can get the browser's language or user-agent using the Navigator object.
```

Comment:
```suggestion
A Navigator object contains information about the browser. For example, you can get the browser's language via the `window.navigator.language` property.
```

I'm thinking a more specific example might be helpful here. Let me know what you think!